### DISCUSSION TITLE
Possible to get Helm-Intellisense integration?

### DISCUSSION THREAD

**Discussion by esn89** [neoclide/coc.nvim#3616] [Extension request]
**Is your feature request related to a problem? Please describe.**
Is it possible to get helm intellisense integrated into coc-nvim?
https://github.com/tim-koehler/Helm-Intellisense

**Describe the solution you'd like**
Would love to be able to work with helm charts and yaml files with intellisense

**Describe alternatives you've considered**
Currently using coc-yaml but it seems to be freaking out at some templates because of curly brackets.

**Comment by r10r** [comment]
@chemzqm I would love to see this too. Maybe I can help.

Can you please give me a hint where to start ? How can https://github.com/tim-koehler/Helm-Intellisense be integrated ?

**Reply by fannheyward** [reply]
After a quick look at Helm-Intellisense, an extension will be better.

**Reply by chemzqm** [reply]
Replace api of VSCode with api from coc.nvim, you can checkout https://github.com/neoclide/coc-vetur for example

**Reply by r10r** [reply]
Thanks! I'll try that.

**Reply by lemonshir** [reply]
Hey guys, thanks a lot for all your excellent work! @r10r Could I know the status of this new feature?
